### ISSUE TITLE
Activate hover event to show icon while tabbing

### ISSUE THREAD

**Issue by bistaastha** [fluid-project/infusion-docs#151]
I ran the current master of infusion docs. While tabbing through the doc pages, the links for headings looked something like this-
![Screenshot from 2020-02-23 08-22-21](https://user-images.githubusercontent.com/40273492/75102716-17fdce00-5616-11ea-97c9-7b5759ce2bf3.png)
It feels out of place here.

Hovering over the heading using the cursor looked something like this:
![Screenshot from 2020-02-23 08-22-30](https://user-images.githubusercontent.com/40273492/75102735-349a0600-5616-11ea-8ca9-00713ce28d2d.png)
While here it is apparent that the link refers to the heading.

Doing something similar to :hover mouse-based event while tabbing would remove the ambiguity from the tab selected link.

**Comment by jhung** [comment]
Thanks for catching that. Do you have a solution planned? We should discuss it before starting a pull request. Thanks!

**Comment by bistaastha** [comment]
I initially thought it might be related to this part of the infusion docs CSS 
https://github.com/fluid-project/infusion-docs/blob/05bba0b6e2509241b13d4b5c50656914b1e30314/src/static/css/infusion-docs.css#L149 .

I tried removing that completely, to see if the :hover still works. Turns out the octicon(the image of the black link) still appears on hovering over it. I am not able to completely understand what's going on.
